src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2008,6 +2036,7 @@ internal void EnsureAnonymousTypeTemplates(CancellationToken cancellationToken)
             IMethodSymbol debugEntryPoint = null,
             Stream sourceLinkStream = null,
             IEnumerable<EmbeddedText> embeddedTexts = null,
+            Stream metadataPeStream = null,
```

Comment:
>metadataPeStream [](start = 19, length = 16)

'e' should be upper case: metadataPEStream

I'd suggest a different name though. Something like contractAssemblyStream, or referenceAssemblyStream.